xbmc/guilib/GUITextureD3D.h

```diff
@@ -21,7 +21,8 @@ class CGUITextureD3D : public CGUITexture
   static void DrawQuad(const CRect& coords,
                        UTILS::COLOR::Color color,
                        CTexture* texture = nullptr,
-                       const CRect* texCoords = nullptr);
+                       const CRect* texCoords = nullptr,
+                       float depth = 1.0);
```

Comment:
Please add the same parameter in GUITextureD3D.cpp so that it builds. Is there a reason it's not const?

Reply:
Done.

The reason of it not being const is due to the lack static typing in vanilla Python :).